.github/sync-repo-settings.yaml

```diff
@@ -27,7 +26,6 @@ branchProtectionRules:
     requiresCodeOwnerReviews: true
     requiresStrictStatusChecks: false
     requiredStatusCheckContexts:
-      - dependencies (8)
```

Comment:
Shall we do this for other patterns too? 4.0.x and 5.2.x.

Reply:
Done.